lib/mayaUsd/fileio/shading/shadingModeExporterContext.h

```diff
@@ -105,18 +110,39 @@ class UsdMayaShadingModeExportContext
     AssignmentVector GetAssignments() const;
 
     /// Use this function to create a UsdShadeMaterial prim at the "standard"
-    /// location.  The "standard" location may change depending on arguments
-    /// that are passed to the export script.
+    /// location and bind it to all assignments provided. The "standard" location
+    /// may change depending on arguments that are passed to the export script.
+    ///
+    /// Note: Call this function if correct UV linking is not a requirement. Use
+    ///       the two-stage version otherwise.
```

Comment:
Are there use cases where you'd prefer **not** to incorporate UV linking? I'm wondering whether there's a strong reason why we should keep `MakeAndBindStandardMaterialPrim()`, or if we should just remove it and always use `MakeStandardMaterialPrim()` followed by `BindStandardMaterialPrim()`. `shadingModePxrRis.cpp` and `mayaToHydra/shadingModeExporter.cpp` seem to be the only usages of it that I can see.